app/views/participations/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with(model: participation) do |form| %>
+  <% if participation.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(participation.errors.count, "error") %> prohibited this participation from being saved:</h2>
+
+      <ul>
+        <% participation.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%# TODO how can I make this enforceable %>
```

Comment:
Make what enforceable?

Reply:
Making set program_id enforceable since currently this is just a hidden field where the value can be changed before submission. I have created issue #3 to address this at a later time